go/cmd/cli/commands/run/run.go

```diff
@@ -0,0 +1,232 @@
+package run
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/unkeyed/unkey/go/apps/api"
+	"github.com/unkeyed/unkey/go/apps/ctrl"
+	"github.com/unkeyed/unkey/go/cmd/cli/cli"
+	"github.com/unkeyed/unkey/go/pkg/clock"
+	"github.com/unkeyed/unkey/go/pkg/tls"
+	"github.com/unkeyed/unkey/go/pkg/uid"
+)
+
+var Command = &cli.Command{
+	Name:  "run",
+	Usage: "Run Unkey services",
+	Description: `Run various Unkey services including:
+  - api: The main API server for validating and managing API keys
+  - ctrl: The control plane service for managing infrastructure`,
+	Commands: []*cli.Command{
+		apiCommand,
+		ctrlCommand,
+	},
+}
+
+var apiCommand = &cli.Command{
```

Comment:
is this like an example of how to use it? cause I'm so confused as to why this is all in a single new file